index.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const socket = require('socket.io');
+const { serverConfig } = require('./src/constants');
+const addNewUser = require('./src/server/addNewUser');
+const sendingMessage = require('./src/server/sendingMessage');
+const userDisconnected = require('./src/server/userDisconnected');
+const userIsTyping = require('./src/server/userIsTyping');
+
+const PORT = serverConfig.port;
+const HOSTNAME = serverConfig.hostname;
+const chatUsers = new Set();
+const app = express();
+
+const server = app.listen(PORT, () => {
+	console.log(`Visit ${HOSTNAME}:${PORT}`);
```

Comment:
I'm pretty sure that you can get host name using `server.address().address` or something like this https://nodejs.org/api/net.html#net_server_address
It would be more suitable for multiple environments